public/webrtc.js

```diff
@@ -293,6 +296,11 @@ function appendMessage(avatar, username, message) {
     chatbox.scrollTop = chatbox.scrollHeight
 }
 
+function isUrl(message) {
+    let regexp = /(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/
```

Comment:
test with this message : 
```
Hello http://perdu.com
```

Reply:
you shoud replace regex : 
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/String/replace